partials/new-header.hbs

```diff
@@ -0,0 +1,100 @@
+<div id="header" class="new-header">
+    <div class="container header">
+        <a href="https://holochain.org" class="logo_link" target="_blank" rel="noopener noreferrer"><img src="{{ @site.logo }}" alt=""></a>
+        <div class="hamburger hamburger--slider js-hamburger">
+            <div class="hamburger-box">
+                <div class="hamburger-inner"></div>
+            </div>
+        </div>
+        <div class="mobile_menu">
+            {{> mobile-nav}}
+        </div>
+        <ul class="main_menu top-nav">
+            <li id="what_is_holochain_item" class="menu_item"><button>WHAT IS HOLOCHAIN</button>
+                <ul class="sub-menu">
+                    <li>
+                        <a href="https://holochain.org/what-holochain"><span>What is Holochain?</span> <span>Framework and P2P networking</span></a>
```

Comment:
```suggestion
                        <a href="https://holochain.org/what-holochain" target="_blank" rel="noopener noreferrer"><span>What is Holochain?</span> <span>Framework and P2P networking</span></a>
```

(see previous comments on the mobile nav file, please also add these to the external links below.)